wagtail/contrib/settings/templates/wagtailsettings/edit.html

```diff
@@ -1,7 +1,7 @@
 {% extends "wagtailadmin/base.html" %}
 {% load i18n wagtailadmin_tags %}
 
-{% block titletag %}{% blocktrans trimmed %}Editing {{ instance }}{% endblocktrans %}{% endblock %}
+{% block titletag %}{% blocktrans trimmed with instance=instance|capfirst %}Editing {{ instance }}{% endblocktrans %}{% endblock %}
```

Comment:
Fyi: I was not 100% sure if I should change the format of the string, but I've kept it, so it won't create a new string id to translate.

Reply:
good call